app/views/shared/_owned_table.html.erb

```diff
@@ -38,7 +38,9 @@
         </td>
         <td>
           <% if child.class == Upload %>
-            <%= link_to "Delete", upload_path(child), method: :delete, class: "btn"%>
+            <%= link_to "Delete", upload_path(child), method: :delete, class: "btn btn-upload-delete"%>
```

Comment:
@AELSchauer do you have thoughts on how to refactor this out of the view? Genuinely curious, I'm a repeat offender at putting if/else logic into views.